### PR TITLE
feat: add pin messages button to edit channel popup

### LINKED ISSUE #2597: Add Pin messages button to the edit channel button
https://user-images.githubusercontent.com/82375995/119331051-94ef7b80-bc8f-11eb-8519-ca47c4a4fbad.png

https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5971%3A42716

### PR DESCRIPTION
**Pull request by richard-ramos**
Fixes #2597

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Do the features of the pin popup work? I know opening the context menu didn't work in the group info popup because it didn't have access to the context menu.

**Comment by richard-ramos** [comment]
> Do the features of the pin popup work?

they do now :D

**Comment by jrainville** [comment]
> they do now :D

Ah yes, that's what I thought 🕵️

File: ui/app/AppLayouts/Chat/ChatColumn.qml [hunk 1]
```diff
@@ -78,10 +78,6 @@ StackLayout {
         }
     }
 
-    StatusImageModal {
-        id: imagePopup
-    }
-
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ChatColumn.qml:84]: Do you still need to move this

**Review by jrainville — APPROVED** [review]

**Comment by anastasiyaig** [comment]
looks great!

**Pull request merged**